subprojects/resources-gcs/src/integTest/groovy/org/gradle/integtests/resource/gcs/GcsClientIntegrationTest.groovy

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.resource.s3
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import org.gradle.integtests.resource.gcs.fixtures.GcsServer
+import org.gradle.internal.credentials.DefaultAwsCredentials
+import org.gradle.internal.resource.transport.gcs.GcsClient
+import org.gradle.test.fixtures.file.TestNameTestDirectoryProvider
+import org.junit.Rule
+import spock.lang.Ignore
+import spock.lang.Shared
+import spock.lang.Specification
+
+class GcsClientIntegrationTest extends Specification {
+
+    public static final String FILE_NAME = "mavenTest.txt"
+    final String accessKey = 'gradle-access-key'
+    final String secret = 'gradle-secret-key'
+    final String bucketName = 'org.gradle.artifacts'
+
+    @Rule
+    final TestNameTestDirectoryProvider temporaryFolder = new TestNameTestDirectoryProvider()
+
+    @Shared
+    GoogleCredential gcsCredentials = new GoogleCredential()
+
+    @Rule
+    public final GcsServer server = new GcsServer(temporaryFolder)
+
+    def setup() {
+        gcsCredentials.refreshToken();
+    }
+
+//    @Unroll
```

Comment:
Should this test be uncommented or deleted? I see that S3ClientIntegrationTest doesn't have the same type of test.